modules/virt-hot-plugging-a-virtual-disk.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+[id="virt-hot-plugging-a-virtual-disk_{context}"]
+= Hot-plugging a virtual disk
+
+Hot-plug virtual disks that you want to attach to a virtual machine instance (VMI) while a virtual machine is running.
+
+.Prerequisites
+* You must have a running virtual machine to hot-plug a virtual disk.
+* You must have at least one data volume or persistent volume claims (PVC) available for hot-plugging.
+
+To hot-plug a virtual disk using the CLI, use the `addvolume` command with `virtctl`.
+
+.Procedure
+
+. Run the following command:
+
+----
+$ virtctl addvolume <virtual-machine|virtual-machine-instance> --volume-name=<datavolume|PVC> [--persist] [--serial=<label-name>]
+----
+
+* The optional `--persist` flag adds the hot-plugged disk to the virtual machine specification as a permanently mounted virtual disk. After running the command with this option:
+** The virtual disk is no longer able to be hot-plugged or hot-unplugged.
+** The virtual disk can no longer be added to or removed from the virtual machine while the virtual machine is running.
++
+[NOTE]
+====
+Specifying the `--persist` flag adds the hot-plugged virtual disk to the virtual machine specification as a permanently mounted virtual disk, but this update does not take effect until the virtual machine is stopped, restarted, or rebooted. Until then, you can hot-plug or hot-unplug the virtual disk. The `--persist` flag applies to virtual machines, not virtual machine interfaces.
```

Comment:
This note and the bullet point above seem contradictory to each other. My suggestion would be to get rid of the note and combine this text with the bullet point text. Also, avoid passive voice.

Reply:
Got rid of the note and replaced with this:

_** Use the optional `--persist` flag to add the hot-plugged disk to the virtual machine specification as a permanently mounted virtual disk. Stop, restart, or reboot the virtual machine to permanently mount the virtual disk. After specifying the `--persist` flag you can no longer hot-plug or hot-unplug the virtual disk. The `--persist` flag applies to virtual machines, not virtual machine interfaces._